app/code/community/Bolt/Boltpay/Helper/DataDogTrait.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2019 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+require_once(Mage::getBaseDir('lib') . DS . 'Boltpay/DataDog/Autoload.php');
+
+trait Bolt_Boltpay_Helper_DataDogTrait
+{
+    public static $boltDataDogKey = '66d80ae8d0278e3ee2d23e65649b7256';
+    private $_apiKey;
+    private $_severityConfig;
+    private $_data = array();
+    private $_datadog;
+
+    /**
+     * Log information
+     *
+     * @param $message
+     * @param array $additionalData
+     * @return DataDog_Client
+     */
+    public function logInfo($message, $additionalData = array())
+    {
+        if ($this->_apiKey && !in_array(DataDog_ErrorTypes::TYPE_INFO, $this->_severityConfig)) {
```

Comment:
IMO `DataDog_Client#log` is better place for this logic; instead of having similar logic in three different method in this class, we can check in log method